Cargo.toml

```diff
@@ -94,13 +95,12 @@ ui-precompiled = [
 ui-enabled = ["dapps"]
 dapps = ["parity-dapps"]
 jit = ["ethcore/jit"]
-dev = ["clippy", "ethcore/dev", "ethcore-util/dev", "ethsync/dev", "parity-rpc/dev", "parity-dapps/dev"]
```

Comment:
Why is `ethcore-util/dev` removed from here?

Reply:
`ethcore-util/dev` does not exist any more

Reply:
Fair enough, anyway see #7355